### ISSUE TITLE
EXOC5 annotation to exocyst

### ISSUE THREAD

**Issue by mcourtot** [geneontology/go-annotation#1426]
Dear InterPro,

We received a user request to add the 'exorcist' annotation to EXOC5. For other components of the complex, we seem to be receiving the mapping via you:
http://www.ebi.ac.uk/interpro/search?q=Exocyst+complex+component

Has EXOC5 been somehow skipped, or is it not suitable for InterPro mapping?

Thanks,
Melanie

**Comment by asangrador** [comment]
Hi Melanie,

We have looked into the exocyst subunits. We didn't have the 'exocyst' annotation for EXOC5 because we didn't have an appropriate entry. This has been solved now; we have created a new entry specific for EXOC5  and mapped it to 'exocyst'. This will be visible from our next release in September. We also noticed that we were missing the annotation for another subunit (exo84), and this has been sorted out too.

Thanks for sending us this request, it helps us to improve our annotation by focusing on a certain complex/area.

Cheers,

Amaia
